src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOOUNT = 20;
+  const PRICE_PER_DAY = 40;
+
+  const basePrice = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return basePrice - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    return basePrice - SHORT_TERM_DISCOOUNT;
+  }
```

Comment:
You can simplify second if like this:

```suggestion
  if (days >= SHORT_TERM) {
    return basePrice - SHORT_TERM_DISCOUNT;
  }
```
because earlier you already checked that the number of days is less than LONG_TERM